### PR TITLE
WIP: adjusted webpack to build to dist/ folder

### PR DESCRIPTION
**Pull request by scramb**
## Description

Adjusted Webpack to use dist/ folder for dev and prod - for dev it just copy's necessary files to serve from, in prod mode it will copy all required files and override all changed files, still would rm -rf before release. 

## Related Issue

N/A

## Motivation and Context

requested in chat

## How Has This Been Tested?

N/A

## Screenshots (if appropriate):
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] adjust makefile to only sign and tar.gz

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
Build failing

**Comment by DeepDiver1975** [comment]
conflicting ..... :fearful:

**Comment by scramb** [comment]
will rebase it as soon as its working will adjust title to WIP

File: src/default.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const _flatten = require('lodash/flatten');
 
 import Phoenix   from './Phoenix.vue';
 
-// --- Adding global libraries ---
+// --- Adding global libraries ---2
```
Comment (DeepDiver1975) [inline src/default.js]: :two: :question:

File: themes/owncloud.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     "error": "#F0506E"
   },
   "logo": {
-    "mini": "core/gfx/cloud-logo-invert.svg",
-    "small": "core/gfx/cloud-logo-invert.svg",
-    "big": "core/gfx/cloud-logo-invert.svg"
+    "mini": "static/gfx/cloud-logo-invert.svg",
```
Comment (DeepDiver1975) [inline themes/owncloud.json]: i guess we should move them into the themes folder ....

**Comment by scramb** [comment]
rebased to #366

**Pull request closed without merging**